src/material-experimental/mdc-card/BUILD.bazel

```diff
@@ -22,13 +22,22 @@ sass_library(
     srcs = glob(["**/_*.scss"]),
     deps = [
         "//src/material-experimental/mdc-helpers:mdc_helpers_scss_lib",
+        "//src/material-experimental/mdc-helpers:mdc_scss_deps_lib",
         "//src/material/core:core_scss_lib",
     ],
 )
 
 sass_binary(
     name = "card_scss",
     src = "card.scss",
+    include_paths = [
+        "external/npm/node_modules",
+    ],
+    deps = [
+        "//src/material-experimental/mdc-helpers:mdc_helpers_scss_lib",
+        "//src/material-experimental/mdc-helpers:mdc_scss_deps_lib",
+        "//src/material/core:all_themes",
```

Comment:
why do we bring in the whole theme here? shouldn't we just fine-grain what is used?

This generally causes unnecessary rebuilds if unrelated things change and the Sass compilation is not very fast right now (it's a bottleneck)

Reply:
Good call, I just copy-pasted from the checkbox without thinking about it. It's actually not necessary at all.